gateway-service-knoxsso/src/main/java/org/apache/knox/gateway/service/knoxsso/WebSSOResource.java

```diff
@@ -269,7 +289,7 @@ private Response getAuthenticationToken(int statusCode) {
       } catch (IOException e) {
         log.unableToCloseOutputStream(e.getMessage(), Arrays.toString(e.getStackTrace()));
       }
-    } catch (TokenServiceException| AliasServiceException e) {
+    } catch (TokenServiceException| AliasServiceException | UnsupportedEncodingException e) {
```

Comment:
Assuming `UnsupportedEncodingException` comes from the `new String`?

Reply:
Right